docs/src/geometries/primitives.md

```diff
@@ -153,3 +153,29 @@ Torus
 ```@example primitives
 Torus((0.,0.,0.), (1.,0.,0.), (0.,1.,0.), 0.2) |> viz
 ```
+
+### ParaboloidSurface
+
+Surface of a paraboloid with the focal axis aligned along the ``z`` direction and clipped up to some radius on the ``xy`` direction.
+
+The equation of the paraboloid is the following:
+
+```math
+f(x, y) = \frac{(x - x_0)^2 + (y - y_0)^2}{4f} + z_0\qquad\text{for } x^2 + y^2 < r^2,
+```
+where ``(x_0, y_0, z_0)`` is the vertex of the parabola, ``f`` is the focal length, and ``r`` is the clip radius. See also <https://en.wikipedia.org/wiki/Paraboloid>.
+
```

Comment:
Let's try to keep the entire docstring in the source code so that we can easily update it in place in the future. No need to add this extra documentation in the docs file directly.
```suggestion
```